tapir/coop/templates/coop/email/resignedmembership_confirmation_subject.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% load utils %}
+{% blocktranslate with coop_share_owner_id=share_owner.id share_owner_first_name=share_owner.first_name share_owner_family_name=share_owner.last_name %}
```

Comment:
If there is a TapirUser linked to the ShareOwner, then the first_name, last_name... fields are empty. You can use `get_display_name_full` that will always get the right field.

Reply:
I think so? I can show you how to see what the emails look like once sent so we can check it.